docs/node/CATEGORIES/02-mining.md

```diff
@@ -55,11 +55,20 @@ interface MiningInfo {
 interface MasternodeInfo {
   masternodeid?: string
   masternodeoperator?: string
-  masternodestate?: 'PRE_ENABLED' | 'ENABLED' | 'PRE_RESIGNED' | 'RESIGNED' | 'PRE_BANNED' | 'BANNED'
+  masternodestate?: MasternodeState
```

Comment:
Helped to change the variable to enum. Remove PRE_BANNED and BANNED as both states are no longer used anymore